telegram/ext/_extbot.py

```diff
@@ -2242,7 +2242,20 @@ async def send_media_group(
         pool_timeout: ODVInput[float] = DEFAULT_NONE,
         api_kwargs: JSONDict = None,
         rate_limit_args: RLARGS = None,
+        group_caption: Optional[str] = None,
+        group_caption_parse_mode: Optional[str] = None,
+        group_caption_entities: Union[List["MessageEntity"], Tuple["MessageEntity", ...]] = None,
     ) -> List[Message]:
+        """Shortcut for::
+
+             await bot.send_media_group(update.effective_chat.id, *args, **kwargs)
+
+        For the documentation of the arguments, please see :meth:`telegram.Bot.send_media_group`.
+
+        Returns:
+            List[:class:`telegram.Message`]: On success, instance representing the message posted.
+
+        """
```

Comment:
This docstring is wrong. please remove.

Reply:
Yes, I saw that that was quite a lonely docstring. Removing. Just for the sake of understanding, though: why is it wrong?

Reply:
Because `Bot.send_media_group` can not be a shortcut for `Bot.send_media_group` ;)

Reply:
I see, I think I understand the logic of the classes a bit better now.